.gitmodules

```diff
@@ -21,3 +21,5 @@
 	path = ext/googletest
 	url = git@github.com:google/googletest.git
 	branch = release-1.12.1
+[submodule "ext/googletest/"]
+	url = https://github.com/google/googletest.git
```

Comment:
please remove ssh/git address and replace with https instead of adding another submodule

Reply:
Interesting ... I was relying on
```
 git submodule set-url ext/googletest/ https://github.com/google/googletest.git
 ```
doing the right thing here.